parts/linux/cloud-init/artifacts/components.json

```diff
@@ -828,7 +828,7 @@
               {
                 "k8sVersion": "1.30",
                 "renovateTag": "OCI_registry=https://mcr.microsoft.com, name=oss/binaries/kubernetes/azure-acr-credential-provider",
-                "latestVersion": "v1.30.0"
+          "latestVersion": "v1.30.7"
```

Comment:
why the different tabulation ?

Reply:
will update the renovate.json to add extra tab

Reply:
closing this PR for Renovate to create another one with correct indent.